### ISSUE TITLE
Make hide button optionally remove injection from message

### ISSUE THREAD

**Issue by DracoClaw** [AAGaming00/rich-quotes#110]
The new "X" button to hide the embeds is great! One issue I am seeing with it, is that it doesn't re-show the message link. So after clicking it there is just an empty message.

**Comment by MulverineX** [comment]
Disable the plugin if you want regular message links, there is a plugin for doing that easily with a command. Not fixing this

**Comment by DracoClaw** [comment]
nonono you misunderstand. I want the message embeds. But if you click the X button, it should put the message link back, not have an empty message.

**Comment by MulverineX** [comment]
@DracoClaw I understand perfectly. I won't be fixing this because it's a pain and this feature is already a niche of a niche. Either you keep the quote or you don't, I'm not implementing a ginormous hack for this. This doesn't even fix anything for Errored quotes because those don't have an x (and won't get one), you can use View Raw to get the underlying link, or again, disable RQ.

**Comment by MulverineX** [comment]
However an "attempt jump" button for Errored quotes might be possible, and is useful, file that as a seperate issue if that's something you want.

**Comment by MulverineX** [comment]
Also if the quote is not Errored you can get the link in the More menu btw, it extends the regular message context menu.

**Comment by DracoClaw** [comment]
This isnt really about errored embeds. The thought process is, if a reallly long message is linked, the embed can be quite large, so being able to hide the embed and just have a link, FOR THAT SPECIFIC MESSAGE, would be helpful. This can also be helpful if lets say a message is linked with contents you dont want to see, maybe its nsfw or its spoilers, but you still want the message link.

**Comment by DracoClaw** [comment]
basically pressing the X should stop the plugin from effecting that message, so it reverts back to showing the link

**Comment by MulverineX** [comment]
Well I could easily make it remove our injection, but then any other quotes in that message would also revert to link form, is that okay? @DracoClaw

**Comment by MulverineX** [comment]
I'd make it a setting between that and current behavior

**Comment by DracoClaw** [comment]
Yeah that could work. Similar to the native "Remove All Reactions".
